var/spack/repos/builtin/packages/duckdb/package.py

```diff
@@ -124,7 +127,7 @@ def patch(self):
                 'set(DUCKDB_VERSION "v{0}")\n\\1'.format(version),
                 "CMakeLists.txt",
             )
-        else:
+        elif self.spec.satisfies("@0.10.2:"):
```

Comment:
Is this the right check since the statements in the `else` clause will now only apply for versions `0.10.2` on?

Reply:
Thanks for pointing that out. That was not correct. In retrospect that process is only needed for version 0.10.0 (and maybe 0.10.1, but that's not included in the package).